gammapy/background/fov.py

```diff
@@ -3,3 +3,60 @@
 Field-of-view (FOV) background estimation
 """
 from __future__ import print_function, division
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+import numpy as np
+from scipy.interpolate import interp1d
```

Comment:
This will fail on travis-ci. `scipy` is an optional dependency, so you should import it in the function where you use it.
The rule in Astropy is that users with missing optional dependencies should be able to import any Astropy / Gammapy module, and then only get the `ImportError` for the optional dependency when the call the function that needs it.